about-page.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Tech Academy</title>
+  <link rel="stylesheet" href="https://use.typekit.net/kjf8squ.css">
+
+  <link rel="stylesheet" href="style.css">
+ 
+</head>
+
+<body>
+  <header class="primary-header">
+    <div class="container">
+      <nav class="navigation">
+    <div class="menu-button"><img id="humberger" src="img\menu.svg" alt="menu-button"></div>
+
+      <a href="index.html"><img class="logo"src="img/logo-2.png" alt="logo" /></a>
+      <div class="nav-bar">
+      <ul  class="nav-list text-neutral-900 fw-bold fs-500">
+      <li><a class="nav-link" href="index.html">Home</a></li>
+      <li><a class="nav-link" href="about-page.html">About</a></li>
+      <li><a class="nav-link" href="#courses">Courses</a></li>
+      <li><a class="nav-link" href="#instructor">Instrictors</a></li>
+    </ul>
+      </div>
+  </nav>
+</div>
+</header>
+<div id="nav-wrap">
+<div id="close-button"><img src="img/x-icon.svg" alt="close-button"></div>
+<div id="nav-m-wrap">
+  <li><a href="index.html">Home</a></li>
+  <li><a href="about-page.html">About</a></li>
+  <li><a href="#courses">Courses</a></li>
+  <li><a href="#instructor">Instrictors</a></li>
+</div>
+</div>
+
+  <main>
+    <section id="about-hero">
+      <div class="container">
+        <div class="hero">
+          <div>
+            <h1 class="fs-secondary-heading fw-bold text-accent-400 ">Break into tech with <br>the tech academy </h1>
+            <div class="about-hero-text">
```

Comment:
### Your Heading
![Screenshot from 2022-10-20 15-35-49](https://user-images.githubusercontent.com/93039359/196926634-2fd60476-b394-48b0-a4d2-4ca3677322e3.png)
### CindyShin Heading
![about-heading](https://user-images.githubusercontent.com/93039359/196926773-b58856cb-010c-4947-b775-39f217d20ba1.png)

- [ ] In the about page you can see that your heading is not very much similar like its font-size and font-weight. Kindly make it similar to the CindyShin design :+1: